tensorboard/plugins/beholder/tf_beholder_dashboard/tf-beholder-dashboard.html

```diff
@@ -32,6 +32,12 @@
 <dom-module id="tf-beholder-dashboard">
   <template>
     <tf-plugin-dialog id="initialDialog"></tf-plugin-dialog>
+    <div class="deprecation-warning">
+      Beholder is deprecated and will be removed in a future TensorBoard
+      release. Please see
+      <a href="https://github.com/tensorflow/tensorboard/issues/3843">#3842</a>
```

Comment:
Text says `#3842`, but you do want #3843.

Reply:
Fixed, thanks for catching that!